drivers/gps/nrf9160_gps/nrf9160_gps.c

```diff
@@ -18,6 +18,9 @@
 #ifdef CONFIG_NRF9160_GPS_HANDLE_MODEM_CONFIGURATION
 #include <modem/lte_lc.h>
 #endif
+#ifdef CONFIG_NRF_CLOUD_AGPS_FILTERED
+#include <net/nrf_cloud_agps.h>
```

Comment:
This include is probably not needed. But the whole GPS driver has been deprecated since v1.7.0 and we're just discussing removing it for v1.9.0. So it's probably not necessary to update the driver anymore.

Reply:
Noted.  As the saying goes, "I'll believe it when I see it." :-)